src/utils/RestfulUtils.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import { ControllerError } from "../enums/ControllerError";
+import { StatusCode } from "../enums/StatusCode";
+import { IRestfulObject } from "../types/BaseResponseApp";
+
+// Utils used by Controllers
+export class RestfulUtils {
+  // Check if token is set into request. If token is invalid and res is provided, an error will be written
+  public static checkTokenIntoRequest(
+    req: express.Request,
+    res?: express.Response
+  ): boolean {
+    if (req.query.token === undefined) {
+      if (res !== undefined) {
+        this.setErrorResponse(res, ControllerError.ERROR_INVALID_TOKEN);
+      }
+      return false;
+    }
+    return true;
```

Comment:
this is not really clear (maybe becuse of the naming). why "checkTokenIntoRequest" returns true in case the token is undefined and res is set ?